docs/validators/validator-setup.md

```diff
@@ -2,10 +2,10 @@
 order: 2
 -->
 
-# Run a Validator on the Cosmos Hub Mainnet
+# Running a Validator
 
 ::: tip
-Information on how to join the mainnet (`genesis.json` file and seeds) is held [in our `launch` repo](https://github.com/cosmos/launch/). 
+We suggest you try out joining a public testnet first. Information on how to join the Vega testnet can be found [here]](https://github.com/cosmos/vega-test/blob/master/public-testnet/README.md). 
```

Comment:
```suggestion
We suggest you try out joining a public testnet first. Information on how to join the Vega testnet can be found [here](https://github.com/cosmos/vega-test/blob/master/public-testnet/README.md). 
```

When we merge the updated joining testnet docs, I might also recommend adding that here or replacing the link

Reply:
Good point, might as well just add it now as we should aim to have the new testnet up asap.

```suggestion
We suggest you try out joining a public testnet first. Information on how to join the most recent testnet can be found [here](../hub-tutorials/join-testnet.md). 
```